smac/smac_cli.py

```diff
@@ -83,12 +85,38 @@ def main_cli(self):
                     fn=traj_fn, cs=scen.cs)
                 initial_configs.append(trajectory[-1]["incumbent"])
 
+        # Restore state
+        stats = None
+        incumbent = None
+        if args_.restore_state:
+            # Check for folder and files
```

Comment:
Could you add some output to debug if restoring fails, e.g. "Trying to restore state from <dir>", "Successfully restored %d runs", "Successfully restored incumbent with value %f"